src/App.tsx

```diff
@@ -1,10 +1,47 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { getpreparedTodos } from './utils/PreparedTodos';
+import { TodosFilter } from './types/TodoFilter';
+import { TodoApp } from './components/TodoApp/TodoApp';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
 
-const USER_ID = 0;
+const USER_ID = 11528;
 
 export const App: React.FC = () => {
+  const [todos, setTodo] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState<TodosFilter>(TodosFilter.All);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  const activeTodosCount = todos
+    .filter(({ completed }) => completed === false).length;
+
+  const completedTodosCount = todos
+    .filter(({ completed }) => completed === true).length;
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(setTodo)
+      .catch((error) => {
+        setErrorMessage('Unable to load todos');
+        // eslint-disable-next-line no-console
+        console.warn(error);
+      });
+    const timerId = setInterval(() => {
+      setErrorMessage('');
+    }, 3000);
```

Comment:
You are cleaning the message every 3000 mSec - looks like it is a bit wrong

Consider using `setTimeout` instead